src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -43,7 +43,7 @@ protected override void ProcessRecord()
     /// default sink (display to console screen)
     /// </summary>
     [Cmdlet(VerbsData.Out, "Default", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113362", RemotingCapability = RemotingCapability.None)]
-    public class OutDefaultCommand : FrontEndCommandBase
+    public class OutDefaultCommand : FrontEndCommandBase, IDisposable
```

Comment:
Do you really need explicitly state `IDisposable` here? `FrontEndCommandBase` already implement `IDisposable` and it is not like you need to reimplement `IDisposable.Dispose()` here.